tests/test_config.py

```diff
@@ -19,6 +19,18 @@
 LOCAL = config.ConfigFile.LOCAL
 ALL = config.ConfigFile.ALL
 
+west_env = {
+    SYSTEM: 'WEST_CONFIG_SYSTEM',
+    GLOBAL: 'WEST_CONFIG_GLOBAL',
+    LOCAL: 'WEST_CONFIG_LOCAL',
+}
+
+west_flag = {
+    SYSTEM: '--system',
+    GLOBAL: '--global',
+    LOCAL: '--local',
+}
```

Comment:
I've been wondering why we don't have more dicts like this in the production code too... but that's for another day!